easybuild/easyconfigs/r/rapidNJ/rapidNJ-2.3.3-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,41 @@
+# This seems to be actively maintained version of the code by the looks of it. 
+# See issue #5
+# https://github.com/somme89/rapidNJ/issues/5
+# why -march=native will not be used
+# Author: J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'MakeCp'
+
+name = 'rapidNJ'
+version = '2.3.3'
+
+homepage = 'https://github.com/somme89/rapidNJ'
+description = """RapidNJ is an algorithmic engineered implementation of canonical 
+neighbour-joining. It uses an efficient search heuristic to speed-up the core 
+computations of the neighbour-joining method that enables RapidNJ to 
+outperform other state-of-the-art neighbour-joining implementations."""
+
+citing = """Rapid Neighbour Joining. Martin Simonsen, Thomas Mailund and Christian N. S. Pedersen.
+In: Proceedings of the 8th Workshop in Algorithms in Bioinformatics (WABI), LNBI 5251, 113-122,
+Springer Verlag, October 2008.
+doi: 10.1007/978-3-540-87361-7_10"""
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+
+source_urls = ['https://github.com/somme89/rapidNJ/archive/']
+sources = [{'filename': 'v%(version)s.tar.gz', 'download_filename': 'latest.tar.gz'}]
+checksums = ['662f864cc3e5bc68aea23129f02e0062cac9ebd37e414b54c5c5e616ff4f245d']
+
+builddependencies = [('binutils', '2.38')]
+
+files_to_copy = [(['bin/rapidnj'], 'bin')]
+
+# That is returning 1 instead of 0, so disabled for now
+# sanity_check_commands = ['rapidnj --help']
```

Comment:
Not worth delaying the merge for this, for you can use an approach like:

`sanity_check_commands = ['olego 2>&1 | grep "version %(version)s"']`

Reply:
Looks fine.